cmd/zed/vacuum/command.go

```diff
@@ -0,0 +1,94 @@
+package vacuum
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/brimdata/zed/cmd/zed/root"
+	"github.com/brimdata/zed/pkg/charm"
+	"github.com/brimdata/zed/pkg/plural"
+)
+
+var Cmd = &charm.Spec{
+	Name:  "vacuum",
+	Usage: "vacuum [options]",
+	Short: "clear space by removing unreferenced objects",
+	Long: `
+"zed vacuum" clears up space in a pool by removing objects that are not visible 
+from a pool's branch or commit.
+
+DANGER ZONE:
+Running this command will permanently delete objects referenced in 
+previous commits causing missing data when time traveling to those commits.
+`,
+	New: New,
+}
+
+type Command struct {
+	*root.Command
+	force  bool
+	dryrun bool
+}
+
+func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	c := &Command{Command: parent.(*root.Command)}
+	f.BoolVar(&c.force, "f", false, "do not prompt for confirmation")
+	f.BoolVar(&c.dryrun, "dryrun", false, "vacuum without deleting objects")
+	return c, nil
+}
+
+func (c *Command) Run(args []string) error {
+	ctx, cleanup, err := c.Init()
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+	at, err := c.LakeFlags.HEAD()
+	if err != nil {
+		return errors.New("default pool and branch unset")
```

Comment:
Every command except for "zed use" just returns the error from LakeFlags.HEAD.
```suggestion
		return err
```